lib/IRGen/GenMeta.cpp

```diff
@@ -4015,6 +4015,23 @@ namespace {
     }
 
     void addIVarDestroyer() {
+      if (IGM.Context.LangOpts.hasFeature(Feature::Embedded)) {
+        llvm::Constant *ptr = nullptr;
+        for (const SILVTable::Entry &entry : VTable->getEntries()) {
```

Comment:
Because `IGM.getAddrOfIVarInitDestroy` only receives the ClassDecl, which is not good enough to emit class functions of *specialized classes* because they all share the same ClassDecl... We have the same problem a couple lines above in `addDestructorFunction`. Suggestions?

Reply:
Ah, I see. This is okay for now!